site/ja/tutorials/load_data/pandas_dataframe.ipynb

```diff
@@ -0,0 +1,488 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "iiyC7HkqxlUD"
+   },
+   "source": [
+    "## pandas を使ってデータを読み込む"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "5IoRbCA2n0_V"
+   },
+   "outputs": [],
+   "source": [
+    "from __future__ import absolute_import, division, print_function, unicode_literals\n",
+    "\n",
+    "try:\n",
+    "  # %tensorflow_version only exists in Colab.\n",
+    "  %tensorflow_version 2.x\n",
+    "except Exception:\n",
+    "  pass\n",
+    "import pandas as pd\n",
+    "import tensorflow as tf"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "heart データセットを含んだCSVをダウンロードします。"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "VS4w2LePn9g3"
+   },
+   "outputs": [],
+   "source": [
+    "csv_file = tf.keras.utils.get_file('heart.csv', 'https://storage.googleapis.com/applied-dl/heart.csv')import"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "6BXRPD2-xtQ1"
+   },
+   "source": [
+    "pandas を使ってCSVを読み込みます。"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "UEfJ8TcMpe-2"
+   },
+   "outputs": [],
+   "source": [
+    "df = pd.read_csv(csv_file)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "8FkK6QIRpjd4"
+   },
+   "outputs": [],
+   "source": [
+    "df.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "_MOAKz654CT5"
+   },
+   "outputs": [],
+   "source": [
+    "df.dtypes"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "dataframe 内で唯一の object 型である 「thal」列を離散値に変換します。"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "LmCl5R5C2IKo"
+   },
+   "outputs": [],
+   "source": [
+    "df['thal'] = pd.Categorical(df['thal'])\n",
+    "df['thal'] = df.thal.cat.codes"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "s4XA1SNW2QyI"
+   },
+   "outputs": [],
+   "source": [
+    "df.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "WWRhH6r4xxQu"
+   },
+   "source": [
+    "## tf.data.Dataset を使ってデータをロードする"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "tf.data.Dataset.from_tensor_slices メソッドを使って、pandas の dataframeから値を読み込みます。\n",
```

Comment:
`tf.data.Dataset.from_tensor_slices` should be emphasized to consistent with original document.